cmake/modules/FindPulseAudio.cmake

```diff
@@ -15,7 +15,7 @@
 #   PulseAudio::PulseAudio   - The PulseAudio library
 
 if(NOT PulseAudio_FIND_VERSION)
```

Comment:
I'm not sure why this block is here. It seems it will allow someone to override the required version with `-DPulseAudio_FIND_VERSION=10.0.0`, Maybe it's better to just remove this whole block and put the version in the pkgconfig check?

Reply:
Thanks much, will do.

Reply:
I force pushed. Did you have that in mind? Disallow overwriting and forcing it in pkg_check?

Reply:
yes, looks fine